api/src/main/java/jakarta/servlet/jsp/el/ImplicitObjectELResolver.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1997, 2021 Oracle and/or its affiliates and others.
+ * Copyright (c) 1997, 2022 Oracle and/or its affiliates and others.
```

Comment:
Why did you update this to 2022, but not the others?

Reply:
Forgetfulness. I need to get back into the habit of updating these. I'll get the commits updated.